scenarios/aws/microVMs/microvms/resources/default/raw_volume.xml

```diff
@@ -0,0 +1,17 @@
+<volume type='file'>
+    <name>{imageName}</name>
+    <key>{volumeKey}</key>
+  <capacity unit='bytes'>104857600</capacity>
+  <allocation unit='bytes'>104857600</allocation>
+  <physical unit='bytes'>104857600</physical>
+  <target>
+      <path>{imagePath}</path>
+      <format type='{format}'/>
+    <permissions>
+      <mode>0660</mode>
+      <owner>$(grep $USER /etc/passwd | cut -d ':' -f 3)</owner>
+      <group>$(grep 'libvirt:' /etc/group | cut -d ':' -f 3)</group>
```

Comment:
```suggestion
      <owner>$(getent passwd $USER | cut -d ':' -f 3)</owner>
      <group>$(getent group libvirt | cut -d ':' -f 3)</group>
```

Reply:
This change was causing a malformed XML error. Left it as is for now.